### ISSUE TITLE
Mapping Jira Cascading fields to seperate fields in ADO

### ISSUE THREAD

**Issue by C-F-Palmer** [solidify/jira-azuredevops-migrator#509]
**Describe the bug**

As referenced in #292 

I am trying to map the cascading fields I have in Jira that looks like this:

![image](https://user-images.githubusercontent.com/92213401/182675300-12a33d17-7d76-4a89-92bc-cc6129203066.png)

![image](https://user-images.githubusercontent.com/92213401/182675402-7a5420c3-bca8-44c7-b432-9ba9b92b3777.png)

I am trying to map them to separate fields in ADO that look like this:

![image](https://user-images.githubusercontent.com/92213401/182676050-21b6cefa-8468-4f2a-90e5-77bf0e7650a0.png)

I have managed to map the "Company" field successfully but cannot seem to get the "Site" field to map to the "Location" field in ADO. I have used the following mapping config:

{
	"source": "customfield_10007",
	"source-type": "id",
	"target":  "Custom.Company"
      },
      {
	"source": "customfield_10007:1",
	"source-type": "id",
	"target":  "Custom.Location"
      },

This information is based on what is returned when I "Inspect" on this filed in Jira via the MS Edge browser:

![image](https://user-images.githubusercontent.com/92213401/182677329-88a88606-411d-41d0-bcc6-7776ef360558.png)

 Thanks

Chris

**Comment by Alexander-Hjelm** [comment]
To my knowledge, we haven't implemented support for cascading picklists as of #292, and no plan to do it in the foreseeable future.

Are you sure that "customfield_10007:1" is a field that lives directly on the object model of the Jira issue? And not some kind of field property? The tool will only migrate FIELDS. You can pull the issue via the REST API to find out for yourself.

**Comment by C-F-Palmer** [comment]
Hi Alexander,

Thanks for the late night response again. Here is an extract from the issue via the API:

"customfield_10007": {
      "self": "https://xxxxxxxxxxx/jira/rest/api/2/customFieldOption/10242",
      "value": "Ikon Science",
      "id": "10242",
      "child": {
        "self": "https://xxxxxxxxxx/jira/rest/api/2/customFieldOption/15020",
        "value": "Support / CCT",
        "id": "15020"

I guess what I am failing to understand is, as you can see from my images, that it looks like two separate fields when you edit them in Jira (second image). 

Is there anything I can do with the above?

**Comment by Alexander-Hjelm** [comment]
Yeah, looks like the missing property is actually a child that lives on the field itself. I'm pretty sure we don't have support for child features today. Since we lack funding for this feature, it will probably not be prioritized. We always welcome community contributions, though.

If it's feasible for you, I would recommend doing an initial migration run with our tool, and writing a custom script for fetching all the supporting data, like child fields, from JIRA, and uploading them to ADO post-migration.

**Comment by Alexander-Hjelm** [comment]
I will mark this as a feature request :)
